stdlib/public/core/FloatingPointParsing.swift.gyb

```diff
@@ -126,36 +126,23 @@ extension ${Self}: LosslessStringConvertible {
   ///   is `nil`.
   @inlinable // FIXME(sil-serialize-all)
   public init?<S: StringProtocol>(_ text: S) {
-    let result: ${Self}? = text.withCString { chars in
-      // TODO: We should change the ABI for
-      // _swift_stdlib_strtoX_clocale so that it returns
-      // a boolean `false` for leading whitespace, empty
-      // string, or invalid character.  Then we could avoid
-      // inlining these checks into every single client.
-      switch chars[0] {
-      case 9, 10, 11, 12, 13, 32:
-        // Reject any input with leading whitespace.
-        return nil
-      case 0:
-        // Reject the empty string
-        return nil
-      default:
-        break
+    self = 0.0
+    let success = withUnsafeMutablePointer(to: &self) { p -> Bool in
+      text.withCString { chars -> Bool in
+        switch chars[0] {
+        case 9, 10, 11, 12, 13, 32:
+          return false // Reject leading whitespace
+        case 0:
+          return false // Reject empty string
+        default:
+          break
+        }
+        let endPtr = _swift_stdlib_strto${cFuncSuffix2[bits]}_clocale(chars, p)
```

Comment:
`_swift_stdlib_strto...` has existed since 5.0, right?

Reply:
oh ignore me, it was there already

Reply:
Yes, no ABI change here.